src/devpanel/panel/index.js

```diff
@@ -18,6 +18,12 @@ var inspectMode = require('api').inspect;
 var inspector = require('../inspector/index.js');
 var currentInspectorName = inspector.currentName;
 
+var KEY_ESC = 27;
+
+inspectBasisDomEvent.captureEvent('keydown', function(event){
```

Comment:
show add global handler on inspect mode on and remove on off
`captureEvent` is a bad choice since the can be a editor in future